charts/k8tz/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: k8tz
 description: Kubernetes admission controller to inject timezones into Pods and CronJobs
-version: 0.11.0
+version: 0.12.0
```

Comment:
don't bump the helm chart version yet, ill release new version soon with these changes